content/sensu-go/5.12/files/allow-list.yaml

```diff
@@ -0,0 +1,14 @@
+- exec: /usr/local/bin/hello.sh
```

Comment:
I wouldn't use this literally, maybe there is a better use case we can exemplify.